.gitignore

```diff
@@ -63,3 +63,6 @@ test-timings.json
 
 # Storybook
 *storybook.log
+
+# Claude
+.claude/settings.local.json
```

Comment:
![shrek-from-the-movie-shrek-is-smiling-and-looking-down](https://github.com/user-attachments/assets/34a08ef3-8da0-4e4b-b5d8-60d2c5e2ff48)